bundles/org.openhab.binding.gree/README.md

```diff
@@ -0,0 +1,147 @@
+# GREE Binding
+
+This binding integrates GREE Air Conditioners.
+
+Note: The GREE Air Conditioner must already be setup on the WiFi network and must have a fixed IP Address.
+
+## Supported Things
+
+This binding supports one Thing type `airconditioner`.
+
+## Discovery
+
+Once the GREE is on the network (WiFi active) it could be discovery automatically.
+An IP broadcast message is sent and every responding units gets added to the Inbox. 
+
+## Binding Configuration
+
+No binding configuration is required.
+
+#### Manual Thing Creation
+
+Fans can be manually created in the *PaperUI* or *HABmin*, or by placing a *.things* file in the *conf/things* directory.  See example below.
+
+
```

Comment:
These sections should be removed.

Reply:
Don't agree. There are many people using textual configuration, so examples are highly appreciated.
Textual config will still remain as an option in openHAB 3.0

Reply:
I think @cpmeister is referring to this paragraph

```
Fans can be manually created in the *PaperUI* or *HABmin*, or by placing a *.things* file in the *conf/things* directory.  See example below.
```

because it's "business as usual"

Reply:
paragraph removed

Reply:
I was also referring to the Binding Configuration section since in our readme template we state:
> If your binding does not offer any generic configurations, you can remove this section completely.